build/versioning.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       arguments: >
         -SwaggerDir '$(System.DefaultWorkingDirectory)/swagger'
         -AssemblyDir '$(System.ArtifactsDirectory)/deploy/webArtifacts/Microsoft.Health.Dicom.Web.dll'
-        -Version 'v1-prerelease'
+        -Version 'v1'
```

Comment:
We should make sure that the documented API version matches the OpenApi version - is it `1` or `1.0`? It makes a difference when hitting the OpenApi url.

Reply:
According to https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#12-versioning it's up to us

Reply:
Done. I have changed the documentation and removed the minor version.